soda/src/main/java/it/okkam/opendata/models/soda/SodaResource.java

```diff
@@ -182,5 +186,37 @@ public String getContact_email() {
   public void setContact_email(String contact_email) {
     this.contact_email = contact_email;
   }
-  
+
+  public String getLens_view_type() {
+    return lens_view_type;
+  }
+
+  public void setLens_view_type(String lens_view_type) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00117)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00100)